frontend/src/pages/GranteeRecord/index.js

```diff
@@ -73,8 +74,30 @@ export default function GranteeRecord({ match, location }) {
     }
   }, [granteeId, match.params, regionId]);
 
+  const baseFilters = [
```

Comment:
Why do we have two sets of filters? Can we just have a single set of filters?

Reply:
Some filters need to be applied the widgets/pages but shouldn't be editable in this Filter Menu component - Grantee & Region ids in this example. There may be another way to do things that's preferable

Reply:
oh good point, forgot about that, this is fine!